examples/azure-mysql-wordpress/porter.yaml

```diff
@@ -4,8 +4,7 @@ mixins:
  
 name: porter-azure-wordpress 
 version: 0.1.0
-invocationImage: deislabs/porter-azure-wordpress:latest
-tag: deislabs/porter-azure-wordpress-bundle:v0.1.0
+tag: getporter/porter-azure-wordpress:v0.1.0
```

Comment:
Now that we aren't under deislabs, we can drop the porter-* prefix. So this should be `getporter/azure-wordpress:v0.1.0`